pkg/network/render.go

```diff
@@ -496,6 +506,29 @@ func renderDefaultNetwork(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.B
 	}
 }
 
+func renderCRDForMigration(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Unstructured, bool, error) {
+	migration := conf.Migration
+	progressing := false
```

Comment:
`progressing` is always false? I see other render methods in this file that don't return any `progressing` so basically it does not seem like we are complying with an interface or harmonizing this render method with the other ones.  So I wonder why we have it.

Reply:
Yes, you're right. I removed it.